mayastor/src/bdev/dev/aio.rs

```diff
@@ -85,6 +89,10 @@ impl CreateDestroy for Aio {
             create_aio_bdev(cname.as_ptr(), filename.as_ptr(), self.blk_size)
         };
 
+        if errno == 0 {
+            uri::dev_set_uuid(&self.name, self.uuid);
```

Comment:
I guess its OK but, dev_set_uuid leaks the device abstraction into the URI abstraction which does not seem to be needed? We can just do something like:

```rust
bdev.lookup(&self.name).map(|b| b.set_uuid(self.uuid))
```
or something like that